<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TablePlotterDialog</class>
 <widget class="QDialog" name="TablePlotterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>917</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Table Plotter</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,4">
     <item>
      <widget class="QWidget" name="sideWidget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="openPushButton">
          <property name="text">
           <string>Open File</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="plotComboBox"/>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>X Axis</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="1" column="2">
            <widget class="QLineEdit" name="minXLineEdit"/>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>min</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>max</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QLineEdit" name="maxXLineEdit"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Y Axis</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>min</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="minYLineEdit"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>max</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="maxYLineEdit"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="seriesScrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>158</width>
             <height>103</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="UtQtGL2DPlotTPWidget" name="plotWidget" native="true"/>
     </item>
    </layout>
   </item>
  </layout>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UtQtGL2DPlotTPWidget</class>
   <extends>QWidget</extends>
   <header>UtQtGL2DPlotWidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
